homeassistant/package_constraints.txt

```
@@ -76,3 +76,7 @@ pandas==1.3.0
 # https://bitbucket.org/mrabarnett/mrab-regex/issues/421/2021827-results-in-fatal-python-error
 # This is fixed in 2021.8.28
 regex==2021.8.28
+
+# anyio has a bug that was fixed in 3.3.1
+# can remove after httpx/httpcore updates its anyio version pin
+anyio>=3.3.1
```

Do we really want > for a critical library. This would mean a new version could be activated without anyone (potentially) having testing it.

```suggestion
anyio==3.3.1
```

---

Well this is a constraint. So as long as it is below what httpcore requires, we're good I think? 

We don't internally rely on this integration.

---

Ahh I misunderstood that, I thought it was part of core.